code/style.css

```diff
@@ -1,58 +1,264 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
-  font-family: "Courier New", Courier, monospace;
+  font-family: 'Open Sans', sans-serif;
+  font-family: 'Prata', serif;
```

Comment:
Won't these two collide with each other? How will elements in the body know which of these two fonts to pick?

Reply:
👍

Reply:
I thought that specifying all the font families I want to use should be specified in the body. Then I can specify which one I want to use in the specific class names? But ok I understand what you mean. So maybe it's better to only specify the font-family within the specific css class, rather than how I've done? @EmOlsz some clarification on how I should do this would be great. Thanks!

Reply:
No, we don't have to specify all font families we are going to use, only the one we want to use.
Moreover, you don't need to add font families to `<body />` only, you can add them to any HTML element